Dataset.py

```diff
@@ -131,8 +131,9 @@ def __init__(self, name=None,
     self.weights = {}
     self.nbytes = 0
     self.num_running_chars = 0  # CTC running chars.
-    self._num_timesteps = 0
+    self._num_timesteps = 0 # TODO: this is only used by the HDFDataset, nowhere else
```

Comment:
(Syntactic note: Space missing before `#`.)
When I search for `_num_timesteps`, I see lots of usages. So your comment is wrong? Or you did not found them? Or what exactly is the question?